### PR TITLE
JNZ3 micors speed-up, 32MHz, Forth OOK receiver

### PR DESCRIPTION
**Pull request by SevenW**

### CODE REVIEW

File: explore/1608-forth/flib/spi/rf69ook.fs [hunk 1]
```diff
@@ -0,0 +1,154 @@
+\ rf69 ook driver
+\ uses spi, rf69
+\ requires constant DIO2 to be defined by boards.fs
+
+  $03 constant RF:BRMSB
```
Comment (SevenW) [inline explore/1608-forth/flib/spi/rf69ook.fs]: Those could be added to rf69.fs

File: explore/1608-forth/jz3/core.fs [hunk 2]
```diff
@@ -8,12 +8,13 @@ include ../flib/spi/rf69.fs
 include ../flib/any/varint.fs
 include ../flib/i2c/bme280.fs
 include ../flib/i2c/veml6040.fs
-include ../flib/i2c/tsl4531.fs
```
Comment (SevenW) [inline explore/1608-forth/jz3/core.fs]: This seems to be unintentional. Fix in the next pull request?

File: explore/1608-forth/flib/spi/rf69ook.fs (hunk 1 above)
Comment (jcw) [inline explore/1608-forth/flib/spi/rf69ook.fs]: Hmm, I don't know which I would prefer - they're not _used_ by the FSK driver...

File: explore/1608-forth/jz3/core.fs (hunk 2 above)
Comment (jcw) [inline explore/1608-forth/jz3/core.fs]: Good, actually - even the veml6040 could be dropped, since the PCB footprint is wrong.

**Comment by jcw** [comment]
Fantastic additions BTW, many thanks!

**Pull request merged**